modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/type/PodcastsSortType.kt

```diff
@@ -60,9 +60,12 @@ enum class PodcastsSortType(
         val default = DATE_ADDED_OLDEST_TO_NEWEST
 
         fun fromServerId(serverId: Int?): PodcastsSortType {
+            if (serverId == null) {
+                return default
+            }
             val sortType = values().firstOrNull { it.serverId == serverId }
             if (sortType == null) {
-                LogBuffer.e(LogBuffer.TAG_INVALID_STATE, "Invalid server ${PodcastsSortType::class.java.simpleName}: $serverId")
```

Comment:
With Proguard this comes out as "k" so I have put in the class name to make it clear.